services/updateUserBlockingStatus.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Created by shujaatali on 2/21/17.
+ */
+exports.updateStatus = function(req,res){
+    var config = require('../nodejs_config/config.js');
+    var db = require('nano-blue')(config.App.CouchServerIp+'/members');
+    db.atomic("cbs", "updateFields", req.body.userId,
```

Comment:
@Shujaat89 we should verify that caller is an admin before proceeding to complete this action. what do u say ?